searx/engines/currency_convert.py

```diff
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
-"""
- currency convert (DuckDuckGo)
+# lint: pylint
+"""currency convert (DuckDuckGo)
+
```

Comment:
What's the point of this change to the docstring? I think previously it looked better.

Reply:
No space before title, then one empty line, then body.

Reply:
Right ... I think the blank line looks weird. And looking at [PEP 257](https://www.python.org/dev/peps/pep-0257/) I don't think it's required ... there is no body.

Reply:
Anything more in this PR?

Reply:
No, otherwise LGTM.

Reply:
OK, fixed blank line.